src/gui/properties/downloadedpiecesbar.cpp

```diff
@@ -28,216 +28,144 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include "downloadedpiecesbar.h"
```

Comment:
Please separate module includes from system/Qt includes and from app includes using blank lines.